docs/user-documentation/datastreams.md

```diff
@@ -1,23 +1,45 @@
+## Media Ownership
 
-![Media tab](../assets/media_use_vocabulary_media_form.png)
+Islandora 8 objects can have any number of media associated with them. Media advertise which object they belong to using a special field,
+"Media Of". By editing this field, you can change the what node owns the media, and therefore, where it gets displayed or managed.
+
+!!! Note
+    The direction of the relationship between objects and datastreams is reversed when compared to Islandora 7.  Generally speaking,
+    objects are unaware of their datastreams, and it's a Drupal view that lists datastreams for an object.
+
+## Media Usage
 
-Because the Media Use vocabulary is an ordinary Drupal vocabulary, Islandora 8 site administrators can add their own tags, and in turn, these local tags can be used to identify media that have some custom local purpose. In fact, the Media Use vocabulary has some tags that identify files that have no real use in Islandora 8 but that are migrated from Islandora 7.x. The "Audit Trail", "Collection Policy", "Dublin Core File", "MODS File", and "RELS-EXT File" all fall into this category. These media are attached to an Islandora 8 object during migrations from Islandora 7.x, so that the content you created in Islandora 7.x can be identified after it has been migrated to Islandora 8, despite the differences between the two platforms.
+Islandora 8 media express their intended use with a special "Media Use" field, which accepts taxonomy terms from the "Media Usage"
+vocabulary. Because the Media Usage vocabulary is an ordinary Drupal vocabulary, Islandora 8 site administrators can add their own
+terms, and in turn, these local terms can be used to identify media that have some custom local purpose.
 
-### Derivatives
+![Media tab](../assets/media_use_vocabulary_media_form.png)
 
-Islandora 8 generates derivatives much like Islandora 7 does. Just as Islandora 7.x's derivatives get assigned datastream IDs, Islandora 8's derivatives are assigned values from the Media Use vocabulary. For example, a thumbnail generated from am image is assigned the "Thumbnail image" tag. The derivatives are listed along with the rest of an Islandora 8 object's media, within the "Media" tab in the object:
+!!! Note
+    Terms from the Media Usage vocabulary are very similar to DSIDs in Islandora 7.  The only difference is that a DSID is immutable,
+    but a media's usage can be changed at any time through the media's edit form.
 
-![Media tab](../assets/media_tab.png)
+## Derivatives
 
-For this image, we can see that a "Service File" and a "Thumbnail Image" derivatives have been generated and added to the object's list of media:
+Islandora generates derivatives based on Media Usage for a Media and the Model of the node that owns it.  All of this is configurable
+using context, but by default, derivatives are generated from "Original Files". When an Original File is uploaded, if the node that
```

Comment:
I'd split to a new paragraph at `, but by default` to separate the general explanation from the provided defaults if using islandora_demo.